packages/pointer_interceptor/pointer_interceptor/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.9.3+7
+
+* Moved pointer_interceptor to subfolder.
```

Comment:
Nit: Moves

(Per https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style)

Reply:
(Also, you don't need to list the package name since this is the changelog for the package.)

I would suggest framing this about the changes that are actually published, rather than the impact for us as maintainers. E.g., "Updates metadata to point to the new source location."